### DISCUSSION TITLE
CyIpOpt: How to impose limit on max. change in design variables?

### DISCUSSION THREAD

**Discussion by Hammad1019** [coin-or/Ipopt#675] [Q&A]
Hi.

I am currently using cyipopt for an optimization simulation. I am interested in understanding whether there is a possibility to impose limit on the max. change in design variables. Could you please provide guidance on how I can define such settings using IPOPT options?

**Comment by svigerske** [comment]
I guess the question is whether one can limit the step that Ipopt is taking. I looked at the options, but couldn't find anything suitable.
